### DISCUSSION TITLE
How to change systemwide GTK overrides

### DISCUSSION THREAD

**Discussion by bunnymnemonic** [SpiralLinux/SpiralLinux-project#344] [Q&A] [answered]
Hi,

One of the GTK themes I would like to use has scrollbars that are wider than 12px. However, the config file (~/.config/gtk-3.0/gtk.css) overrides this setting by forcing all the scrollbars to have a 12px width. I managed to change this setting for locally installed themes but the themes that are installed in usr/share/themes (for apps that start as root, for example Synaptic) do not change.

Could you tell me where I can find the css file to modify?

Thank you for this amazing distribution, by the way!

**Comment by geckolinux** [comment, marked as answer]
Hi there, the themes in `/usr/share/themes/` are unmodified, as the modifications would just get overwritten during system updates. The same SpiralLinux tweaks to the GTK behavior that are found in `~/.config/gtk-3.0/` and `~/.config/gtk-4.0/` are also present under `/root/.config/` , so that's what you would need to change for apps running as root.

**Reply by bunnymnemonic** [reply]
I found it! Thank you very much!
